lisp/ess-r-completion.el

```diff
@@ -367,6 +367,29 @@ To be used instead of ESS' completion engine for R versions >= 2.7.0."
 ;;; COMPANY
 ;;; http://company-mode.github.io/
 
+(defvar company-R-library--cache ())
+(defun company-R-library-all-completions ()
+  (unless company-R-library--cache
+    (let ((proc (ess-get-next-available-process)))
+      (when proc
+        (setq company-R-library--cache
+              (ess-get-words-from-vector
+               "local({ out <- try({rownames(installed.packages())}); print(out, max=1e6) })\n")))))
```

Comment:
I think this is very slow on platforms with huge number of libraries, especially if lib paths are located on network drives. This is the case for @mmaechler  for example.